bag_replicator/src/main/scala/uk/ac/wellcome/platform/archive/bagreplicator/storage/azure/BlobRangeUtil.scala

```diff
@@ -0,0 +1,39 @@
+package uk.ac.wellcome.platform.archive.bagreplicator.storage.azure
+
+import java.util.Base64
+
+import com.azure.storage.blob.models.BlobRange
+
+object BlobRangeUtil {
+  def getRanges(length: Long, blockSize: Long): Seq[BlobRange] =
+    ((0L to length - 1) by blockSize)
```

Comment:
i didn't know you could do this! Nice

Reply:
Yeah, new to me too! Much nicer than the icky Enumeration code I wrote elsewhere.